db/migrate/20150307020451_add_unique_url_to_dinners.rb

```diff
@@ -0,0 +1,5 @@
+class AddUniqueUrlToDinners < ActiveRecord::Migration
+  def change
+    add_column :dinners, :unique_url, :string
```

Comment:
1. Is `NULL` a valid value for this field in the database?  If not, add a `null: false` constraint.
2. Does every `Dinner` need to have a unique value for `unique_url`?  The name implies it does, but there is no uniqueness validation at the ActiveRecord level and no uniqueness constraint at the database level.